checkST2110.js

```diff
@@ -859,14 +1043,17 @@ const test_30_62_1 = sdp => {
   if (hasRtpmap === false && payloadType >= 0) {
     errors.push(new Error(`Line ${lines.length}: For stream ${streamCount}, required attribute 'rtpmap' is missing for audio stream.`));
   }
+  if (params.verbose && errors.length == 0) {
+    console.log('Test Passed: ST 2110-30 Section 6.2.1 Test 1 - Compliant with AES-67 Section 7.1');
+  }
   return errors;
 };
 
-// Test ST 2110-30 Section 6.2.1 Test 2 - Valid audio SDP
-const test_30_62_2 = sdp => {
+// ST 2110-30 Section 6.2.1 Test 2 - Valid audio SDP
+const test_30_62_2 = (sdp, params) => {
```

Comment:
Let's not worry about this duplication for now!

```suggestion
// ST 2110-30 Section 6.2.1 Test 2 - Valid audio SDP
// Note: this is almost identical to test_10_62_1 used for video SDP
const test_30_62_2 = (sdp, params) => {
```

Reply:
ok